Formula/kdoctor.rb

```diff
@@ -0,0 +1,26 @@
+class Kdoctor < Formula
+  desc "Environment diagnostics for Kotlin Multiplatform Mobile app development"
+  homepage "https://github.com/kotlin/kdoctor"
+  url "https://github.com/Kotlin/kdoctor/archive/refs/tags/v0.0.2.tar.gz"
+  sha256 "e95af8f76fb3a6240758af76b5528e57b41793065354c81bff2fbebe435f22df"
+  license "Apache-2.0"
+  head "https://github.com/Kotlin/kdoctor.git", branch: "master"
+
+  depends_on "gradle"
+  depends_on "openjdk"
```

Comment:
It looks like you have an Xcode dependency. Not sure if it's also needed at runtime.

Reply:
Correct, I've added a build dependency